docs/tutorials/template.md

```diff
@@ -4,12 +4,12 @@ linkTitle: "Menu title of the tutorial"
 weight: 90
 type: "docs"
 description: "Short description of your tutorial (1 sentence)."
-# Use either image or GIF/video
-# image: "path to preview image"
+# If GIF+video is available use those - otherwise use an image
+# images: ["path to preview gif if available and less than 1MB in size - otherwise path to preview image"]
+# webmSrc: "path to preview video - ideally in 4:3 format"
+# mp4Src: "path to preview video - ideally in 4:3 format"
+# image: "path to preview image - ideally in 4:3 format"
```

Comment:
If it’s an image, do you need it both here and up there in “images”?

Reply:
yes, needs to be in both places